src/Mobs/Monster.cpp

```diff
@@ -1176,33 +1179,44 @@ std::unique_ptr<cMonster> cMonster::NewMonsterFromType(eMonsterType a_MobType)
 
 			return cpp14::make_unique<cHorse>(HorseType, HorseColor, HorseStyle, HorseTameTimes);
 		}
+		case mtZombieVillager:
+		{
+			int Profession = Random.RandInt(6);
+			if (Profession == 6)
+			{
+				// Give farmers a better chance of spawning
+				Profession = 0;
```

Comment:
Maybe a `std::piecewise_constant_distribution` (http://en.cppreference.com/w/cpp/numeric/random/piecewise_constant_distribution) is better suited.

Reply:
You mean [`std::discrete_distribution`](http://en.cppreference.com/w/cpp/numeric/random/discrete_distribution).

Reply:
Yep, didn't see this nice distribution ;D.